app/package.json

```diff
@@ -226,7 +226,6 @@
         "@jupyter-notebook/notebook-extension": true,
         "@jupyter-notebook/terminal-extension": true,
         "@jupyterlab/application-extension": [
-          "@jupyterlab/application-extension:commands",
```

Comment:
This extension is not enabled anymore, except for the `tree` view, because it already uses the `Ctrl + B` shortcut.
AFAIK the commands in this extension may not be necessary for the Notebook view (not sure it is necessary at all, most of them requires a `LabShell` to work).